src/Umbraco.Core/Composing/IComponent.cs

```diff
@@ -1,28 +1,45 @@
 namespace Umbraco.Cms.Core.Composing;
 
 /// <summary>
-///     Represents a component.
+/// Represents a component.
 /// </summary>
 /// <remarks>
-///     <para>Components are created by DI and therefore must have a public constructor.</para>
-///     <para>
-///         All components are terminated in reverse order when Umbraco terminates, and
-///         disposable components are disposed.
-///     </para>
-///     <para>
-///         The Dispose method may be invoked more than once, and components
-///         should ensure they support this.
-///     </para>
+/// <para>
+/// Components are created by DI and therefore must have a public constructor.
+/// </para>
+/// <para>
+/// All components are terminated in reverse order when Umbraco terminates, and disposable components are disposed.
+/// </para>
+/// <para>
+/// The Dispose method may be invoked more than once, and components should ensure they support this.
+/// </para>
 /// </remarks>
```

Comment:
This is the same XML doc as `IAsyncComponent`, so we can replace it with:
```suggestion
/// <inheritdoc />
```